package/MDAnalysis/coordinates/PDB.py

```diff
@@ -493,6 +495,10 @@ class PDBWriter(base.WriterBase):
     are not set (:code:`None` or :code:`np.zeros(6)`,
     see Issue #2698).
 
+    When atom's record_type attribute is present (i.e. Universe object was
```

Comment:
I would probably say here:
```suggestion
    When the record_types attribute is present (e.g. Universe object was
```

The clarification for e.g. over i.e. is that a user could technically just assign record type themselves.

Reply:
Done